app/controllers/api/user_blocks/active_lists_controller.rb

```diff
@@ -0,0 +1,23 @@
+module Api
+  module UserBlocks
+    class ActiveListsController < ApiController
+      before_action :disable_blocks_check
```

Comment:
Why use flash to smuggle the skip blocks flag to `setup_user_auth`? Could we not just pass an argument to the `authorize` callback and have that pass it on to `setup_user_auth`?

Reply:
I made it similar to [disable_terms_redirect](https://github.com/openstreetmap/openstreetmap-website/blob/b44399fb7c1747a811ea613e7782f77849a6d2e1/app/controllers/api/users_controller.rb#L52-L57)

Reply:
Updated to match #5658